benchmarks/google_size.proto

```diff
@@ -1,7 +1,10 @@
+syntax = "proto2";
```

Comment:
Without it, protoc issues a warning. Given that it *is* a proto2 file, it felt reasonable to be explicit. I was somewhat surprised that it was still unspecified, to be honest...